### ISSUE TITLE
Rotate a linked list.

### ISSUE THREAD

**Issue by sat5297** [sat5297/hacktober-coding#144]

**Comment by matthew-smith0** [comment]
Hello, is this issue still active ?

**Comment by sat5297** [comment]
@matthew-smith0 yeah u can go ahead and solve it.

**Comment by matthew-smith0** [comment]
Could you assign me to this issue thanks

**Comment by sat5297** [comment]
@matthew-smith0  yeah go ahead. and you can solve any number of the active issues u want.

**Comment by matthew-smith0** [comment]
I am sorry to bother you with this, but im doing a work on software evolution. One of my required objectives is to be assigned to an issue and after that have a successful merge. And i need to put it in to my work, that i have been assigned to the issue i found. Thank you.

**Comment by sat5297** [comment]
@matthew-smith0 if u can submit the code to solve the issue and attach a screen shot of code execution. I am happy to have it merged.
